README.md

```diff
@@ -16,7 +16,7 @@ For more background on MCR:
 
 ## Providing Feedback
 
-Please use this repo to provide your feedback on usage or concerns with MCR.
+Please [open issues](https://github.com/microsoft/containerregistry/issues) to provide your feedback on usage, suggestions or concerns with MCR.

```

Comment:
"provide your feedback on usage, as well as any suggestions or concerns with MCR" would read better. As it stands now, it feels like feedback on usage or feedback on suggestions.